packages/opentelemetry-resources/test/detectors/node/BrowserDetector.test.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as sinon from 'sinon';
+import { Resource } from '../../../src';
+import { browserDetector } from '../../../src/detectors/BrowserDetector';
+import {
+  assertEmptyResource,
+} from '../../util/resource-assertions';
+
+
+describe('browserDetector()', () => {
+  beforeEach(() => {
+    (globalThis.window as {}) = {};
```

Comment:
Since this file will be tested in the Node.js environment only, we don't need to mock the `globalThis.window` anymore.

Reply:
fixed